### PR TITLE
S3t2 match guess

### PR DESCRIPTION
**Pull request by Steph-ski**
transform user input and arrays values into uppercase

match the user guess against the given array of values (filter)

add unique correctly guess counties to a new array

clear the input when the correct guess is completed

### CODE REVIEW

File: guessInput.js [hunk 1]
```diff
@@ -1,5 +1,88 @@
...
+}
+
+//handles key presses in the guess box
 const guessHandler = () => {
-    console.log(guessInput.value)
+    let guessInputArray = [guessInput.value]
+    let upperGuessInput = guessInputArray.map(makeUpper)
+//filter function loops through user guesses and compares to counties list. Adds to new array if correct.
+    let correctInputArray = (upperGuessInput.filter((answer) => {
+        let match = false
```
Comment (HadertS) [inline guessInput.js]: lines 11-22 can be replaced with:
`    let correctInputArray = (upperGuessInput.filter((answer) => {
        return upperCounties.includes(answer)
    }))`

**Review by HadertS — CHANGES_REQUESTED** [review]
One small change using the include function for the filter

**Review by randallcoding — APPROVED** [review]
good work

**Review by randallcoding — APPROVED** [review]

**Comment by Yas-Shen** [comment]
Think it's good.

**Review by HadertS — APPROVED** [review]
looks good

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    box-shadow: 0 5px 5px rgba(15, 15, 51, 0.4);
+}
+
+input:not(:placeholder-shown) {
+    margin-bottom: 10px;
+    border-bottom: 6px solid #3782F7;
+    height: 69px;
+}
+
+/*.listContainer {*/
```
Comment (Steph-ski) [inline styles.css:128]: if this is not needed, should be deleted?

File: guessInput.js [hunk 3]
```diff
@@ -1,14 +1,90 @@
...
 let answersContainer = document.querySelector("#answersContainer")
 
 //forEach through correctAnswers array put each answer into list in html
-correctAnswers.forEach((item) => {
-    const listItem = document.createElement("li")
-    listItem.innerText = item
-    answersContainer.appendChild(listItem)
-})
\ No newline at end of file
+// correctAnswers.forEach((item) => {
```
Comment (Steph-ski) [inline guessInput.js:85]: this needs to be deleted

**Review by HadertS — APPROVED** [review]

**Pull request merged**